core/client/src/main/java/alluxio/client/file/options/CompleteUfsFileOptions.java

```diff
@@ -37,15 +35,19 @@
    * default file mode.
    *
    * @return the default {@link CompleteUfsFileOptions}
-   * @throws IOException if failed to set owner from login module
    */
-  public static CompleteUfsFileOptions defaults() throws IOException {
+  public static CompleteUfsFileOptions defaults() {
     return new CompleteUfsFileOptions();
   }
 
-  private CompleteUfsFileOptions() throws IOException {
-    mOwner = SecurityUtils.getOwnerFromLoginModule();
-    mGroup = SecurityUtils.getGroupFromLoginModule();
+  private CompleteUfsFileOptions() {
+    if (SecurityUtils.isSecurityEnabled()) {
```

Comment:
would it make sense to push the if logic into the utility method? I would expect that this pattern will be true for every caller at which point it makes sense to encapsulate it